tests/topotests/bgp_rpki_topo1/r2/bgp_table_rmap_rpki_notfound.json

```diff
@@ -0,0 +1,7 @@
+{
+  "routerId": "192.0.2.2",
+  "defaultLocPrf": 100,
+  "localAS": 65002,
+  "routes": {
```

Comment:
I believe this is always a true match. I mean despite there being any routes under `routes`.

Reply:
I do that to avoid the issue:

```
    expected_nb = len(expected.get("routes"))
    output_nb = len(output.get("routes", {}))

    if expected_nb != output_nb:
        return {"error": "expected {} prefixes. Got {}".format(expected_nb, output_nb)}
```